src/test/java/org/broadinstitute/hellbender/utils/UtilsUnitTest.java

```diff
@@ -978,4 +981,78 @@ public void testTestFilterCollectionByExpressions(Set<String> values, Collection
         Set<String> actual = Utils.filterCollectionByExpressions(values, filters, exactMatch);
         Assert.assertEquals(actual, expected);
     }
+
+    @DataProvider(name = "concatMultipleByteArraysData")
+    public Object[][] concatMultipleByteArraysData() {
+        return new Object[][]{
+                // expected, arrays
+                new Object[]{ ArrayUtils.EMPTY_BYTE_ARRAY, new byte[] { } },
+                new Object[]{ new byte[] {100, 100}, new byte[] {100, 100} },
+                new Object[]{ new byte[] {100, 100}, new byte[] {100, 100}, new byte[] { } },
+                new Object[]{ ArrayUtils.EMPTY_BYTE_ARRAY, new byte[] { }, new byte[] { } },
+                new Object[]{ new byte[] {100, 100, 100, 100}, new byte[] {100, 100}, new byte[] {100, 100} },
+        };
+    }
+
+    // expected listed first to accomodate varargs arrays having to be last parameter
+    @Test(dataProvider = "concatMultipleByteArraysData")
+    public void testConcatMultipleByteArrays(byte[] expected, byte[] ... arrays) {
+        Assert.assertEquals(Utils.concat(arrays), expected);
+    }
+
+    @Test
+    public void testConcatMultipleWithNoArrays(){
+        Assert.assertEquals(Utils.concat(), ArrayUtils.EMPTY_BYTE_ARRAY);
+    }
+
+    @DataProvider(name = "concatTwoByteArraysData")
+    public Object[][] concatTwoByteArraysData() {
+        return new Object[][]{
+                // arr1, arr2, expected
+                new Object[]{ new byte[] { }, new byte[] { }, ArrayUtils.EMPTY_BYTE_ARRAY },
+                new Object[]{ new byte[] {100, 100}, new byte[] { }, new byte[] {100, 100} },
+                new Object[]{ new byte[] { }, new byte[] {100, 100}, new byte[] {100, 100} },
+                new Object[]{ new byte[] {100, 100}, new byte[] {100, 100}, new byte[] {100, 100, 100, 100} },
```

Comment:
Comments on the test cases for the two-arg version of concat():

- As above, the values inside your arrays should not all be the same (100), otherwise you are not actually testing whether the method is preserving the original order of the values correctly during concatenation.
- Include at least one additional test case involving an array of length 1, as that is a common edge case for methods like this.